src/tests/Common/external/external.csproj

```diff
@@ -12,9 +12,9 @@
          This RID value doesn't really matter, since the assets we are copying are not RID specific, so defaulting to Windows here
     -->
     <OutputPath>$(TargetingPackPath)</OutputPath>
-    <TargetFramework>$(NetCoreAppCurrent)</TargetFramework>
-    <NuGetTargetMoniker>$(NetCoreAppCurrentTargetFrameworkMoniker)</NuGetTargetMoniker>
-    <NuGetTargetMonikerShort>$(NetCoreAppCurrent)</NuGetTargetMonikerShort>
+    <TargetFramework>net5.0</TargetFramework>
+    <NuGetTargetMoniker>$(NetCoreAppCurrentIdentifier),Version=v5.0</NuGetTargetMoniker>
+    <NuGetTargetMonikerShort>net5.0</NuGetTargetMonikerShort>
```

Comment:
Why is this necessary? Also why are `NuGetTargetMoniker` and `NuGetTargetMonikerShort` properties set? I would not expect them to have any meaning anymore.